frontend/src/pages/RunDetails.tsx

```diff
@@ -208,8 +229,28 @@ class RunDetails extends Page<RunDetailsProps, RunDetailsState> {
                       </div>
                     </React.Fragment>)}
                   </SidePanel>
+
+                  <div className={css.footer}>
+                  <div className={commonCss.flex}>
+                    <InfoIcon style={{ color: color.lowContrast, height: 16, width: 16 }} />
```

Comment:
nit: move to class?

Reply:
Done. Put it in `Css.tsx` because it's also used in `PipelineDetails`